polygon_cli/actions/diff.py

```diff
@@ -3,9 +3,10 @@
 
 
 def process_diff(options):
+    # resolve paths before load_session changes working directory
     input_files = list(map(os.path.abspath, options.file))
 
-    if not load_session():
+    if not load_session(verbose=options.verbose):
```

Comment:
I think it's not good solution to copy & paste this option support to all action. Maybe just process options to load_session? It will be useful for future common options.